src/main/java/com/rockset/client/ApiClient.java

```diff
@@ -898,6 +899,7 @@ public <T> T handleResponse(Response response, Type returnType) throws Exception
 
                     final ObjectMapper objectMapper = new ObjectMapper();
                     objectMapper.configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS, true);
+                    objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
was this reverted before? I think this was done in some prior release as well

Reply:
I believe the issue is that we made this change previously in https://github.com/rockset/rockset-java-client/commit/8769bcccb299de694b63e4484b94f1dd7b4798ea , but this file (and ErrorModel) is autogenerated, and they got reverted when we re-generated them in https://github.com/rockset/rockset-java-client/pull/44/files#diff-13d9b90d04e3f6dec37fa4bfd06402873afbbd1d553a9feb8897b404b43b7951 .

The best way is to update the code generator with the line instead of making the modification here.

Reply:
yeah the code generator was actually updated https://github.com/rockset/swagger-codegen/commit/466c6c449990c285a2b5cbd67762a1afec30a340. Thus, I am confused

Reply:
same for ErrorModel -> https://github.com/rockset/swagger-codegen/commit/5dd01291c7db2fdcf72bbfcf67b1032691816732